### DISCUSSION TITLE
YT-Spammer-Purge GUI [Update]

### DISCUSSION THREAD

**Discussion by realashleybailey** [ThioJoe/YT-Spammer-Purge#785] [General]
An update on my project for a GUI version of this software, it's coming along well, alot of the GUI design is designed with the ability to quickly and easily slot in new features, I am now working on the onboarding experience where a user will make there first pull down of comments, the comments will then be saved into a Firebase Firestore database and will sit there until a Firebase Function calls a backend service running a mixture of PHP and Python to analyze the comments using YT-Spammer-Purge and create a result log that will be uploaded to the Firestore database allowing users to scroll through and analyze the Spam Comments and either mass delete them, select users and comments that should be whitelisted, or select users that should be blacklisted and individually delete comments. The entire project has been programmed in a way that in the future it can be very easily adapted to use local storage and local API keys to run in an Electron App locally on a users computer. I have attached some previews below.

[Edit]
Right now I have just been calling my software YT-Spammer-Purge GUI however I would love to give it its own unique name so if you have any ideas please leave them below.
<p float="left">
<img src="https://user-images.githubusercontent.com/16636012/161876505-92b8a997-8d71-4e72-9b38-c6365bdafd49.png" height="400px"/>
<img src="https://user-images.githubusercontent.com/16636012/161876511-20a1a8f3-a41f-43c3-9f1a-e0992e46c7c1.png" height="400px"/>
<img src="https://user-images.githubusercontent.com/16636012/161876512-61b417a2-0b77-4a58-a42a-d53f353ce520.png" height="400px"/>
<img src="https://user-images.githubusercontent.com/16636012/161876515-c608776a-59fe-4219-9427-6c73a968bc23.png" height="400px"/>
<img src="https://user-images.githubusercontent.com/16636012/161876518-d61d137d-32a1-4ea8-bf8f-7dfccf52b07c.png" height="400px"/>
</p>

<p float="left">
<img src="https://user-images.githubusercontent.com/16636012/162332362-685b9e68-ebb0-44bd-9963-8808f6463722.png" height="400px;">
<img src="https://user-images.githubusercontent.com/16636012/162332367-9a09c82d-11e3-47c4-853f-6d010ed32913.png" height="400px;">
<img src="https://user-images.githubusercontent.com/16636012/162332369-2446012f-e359-4633-acfa-1dd4cd3dc5b3.png" height="400px;">
<img src="https://user-images.githubusercontent.com/16636012/162332372-c3e0c4bc-25f9-4000-8d24-26c1a5c9a99d.png" height="400px;">
<img src="https://user-images.githubusercontent.com/16636012/162332375-d4cda5af-30dd-4892-baad-1c3d945c5772.png" height="400px;">
</p>

**Comment by TechStudent10** [comment]
While I think it looks nice, some of the features are kind of useless. For example, showing the most popular comment and most commented features. This program has one purpose, and it should do that one thing good. You still haven't shown the main thing this program is for, purging spam comments. Also, are you coding this for a phone? And why are the screenshots humongous?

**Reply by realashleybailey** [reply]
The site is responsive to both Desktop and Mobile, I will update the post with desktop screenshots, Not sure why the screenshots are so big I made the post through the Github App so it probably didn't size properly. The most popular comment section is there mainly for testing the API calls, I am still programming the backend right now so don't have a dataset of spam comments to display yet, the site will also have the ability to moderate and track normal comments to help users interact with viewers more. I will doing another update post towards the end of April of a working model of scanning spam comments and deleting them.

**Reply by TechStudent10** [reply]
Alright 👍🏾
